internal/broker/broker.go

```diff
@@ -564,16 +563,14 @@ func (b *Broker) updateSession(sessionID string, session sessionInfo) error {
 // authCachedInfo represents the token that will be saved on disk for offline authentication.
 type authCachedInfo struct {
 	Token      *oauth2.Token
-	AcquiredAt time.Time
```

Comment:
that wasn’t used anymore and you hide some cleanup here? tsssss tsssss :p

Reply:
This never happened, you never saw it, it was never here... 
[Jedi mind erasing movements]